README.md

```diff
@@ -70,82 +112,141 @@ Docker is needed to run the project in a containerized environment. Follow the a
 Ensure `docker-compose` is installed. Refer to the [Docker Compose installation guide](https://docs.docker.com/compose/install/).
 
 ### Setting Up Environment Variables
-Create or update the `.env` files in the `fastapi_backend` and `nextjs-frontend` directories with the following variables:
 
 **Backend (`fastapi_backend/.env`):**
-
-For local setup:
-```
-DATABASE_URL=postgresql+asyncpg://postgres:password@localhost:5432/mydatabase
-```
-For Docker setup, replace localhost with db:
-```
-DATABASE_URL=postgresql+asyncpg://postgres:password@db:5432/mydatabase
-```
-Other environment variables remain unchanged:
-
-```env
-ACCESS_SECRET_KEY=<your-secret-key>
-RESET_PASSWORD_SECRET_KEY=<your-secret-key>
-VERIFICATION_SECRET_KEY=<your-secret-key>
-OPENAPI_OUTPUT_FILE=../nextjs-frontend/openapi.json
-
-MAIL_USERNAME=<your-username>
-MAIL_PASSWORD=<your-password>
-MAIL_FROM=<your-from>
-MAIL_SERVER=<your-server>
+Copy the `.env.example` files to `.env` and update the variables with your own values.
+```bash
+cp project_name/.env.example project_name/.env
 ```
+1. You will only need to update the User Secrets. You can use the following command to generate a new secret key:
+   ```bash
+   python3 -c "import secrets; print(secrets.token_hex(32))"
+   ```
+2. The DATABASE, Email, OPENAPI, and FRONTEND_URL settings are ready to use locally.
+3. You can check the .env.example file for more information about the variables.
 
 **Frontend (`nextjs-frontend/.env.local`):**
-```env
-NEXT_PUBLIC_API_BASE_URL=http://localhost:8000
-OPENAPI_OUTPUT_FILE=openapi.json
+Copy the `.env.example` files to `.env`. These values are unlikely to change, so you can leave them as they are.
+```bash
+cp project_name/.env.example project_name/.env
```

Comment:
The cp command is wrong. There's no project_name here.
I think this should be:
```
cd fastapi_backend
cp .env.example .env
```

Also fix the frontend (see below)

Reply:
When you use the template you have the option to set the project name. But, I can just change as you suggested.